generated/api.github.com.json

```diff
@@ -415,13 +413,8 @@
         "PUT /repos/{owner}/{repo}/issues/{issue_number}/lock"
       ]
     },
-    "keys": {
-      "url": "https://docs.github.com/en/free-pro-team@latest/rest/reference/permissions-required-for-github-apps/#permission-on-keys",
-      "read": ["GET /user/keys", "GET /user/keys/{key_id}"],
-      "write": ["DELETE /user/keys/{key_id}", "POST /user/keys"]
-    },
```

Comment:
there are likely different permissions for `git_ssh_keys` and `gpg_keys` but we need to verify or get the OpenAPI schema updated